content/en/docs/contribute/participate/_index.md

```diff
@@ -0,0 +1,121 @@
+---
+
+<!-- overview -->
+
+SIG Docs is one of the
+[special interest groups](https://github.com/kubernetes/community/blob/master/sig-list.md)
+within the Kubernetes project, focused on writing, updating, and maintaining
+the documentation for Kubernetes as a whole. See
+[SIG Docs from the community github repo](https://github.com/kubernetes/community/tree/master/sig-docs)
+for more information about the SIG.
+
+SIG Docs welcomes content and reviews from all contributors. Anyone can open a
+pull request (PR), and anyone is welcome to file issues about content or comment
+on pull requests in progress.
+
+You can also become a [member](/docs/contribute/participating/roles-and-responsibilities/#members),
+[reviewer](/docs/contribute/participating/roles-and-responsibilities/#reviewers), or [approver](/docs/contribute/participating/roles-and-responsibilities/#approvers). These roles require greater
+access and entail certain responsibilities for approving and committing changes.
+See [community-membership](https://github.com/kubernetes/community/blob/master/community-membership.md)
+for more information on how membership works within the Kubernetes community.
+
+The rest of this document outlines some unique ways these roles function within
+SIG Docs, which is responsible for maintaining one of the most public-facing
+aspects of Kubernetes -- the Kubernetes website and documentation.
+
+
+
+<!-- body -->
+
+## SIG Docs chairperson
+
+Each SIG, including SIG Docs, selects one or more SIG members to act as
+chairpersons. These are points of contact between SIG Docs and other parts of
+the Kubernetes organization. They require extensive knowledge of the structure
+of the Kubernetes project as a whole and how SIG Docs works within it. See
+[Leadership](https://github.com/kubernetes/community/tree/master/sig-docs#leadership)
+for the current list of chairpersons.
+
+## SIG Docs teams and automation
+
+Automation in SIG Docs relies on two different mechanisms for automation:
+GitHub groups and OWNERS files.
+
+### GitHub groups
+
+The SIG Docs group defines two teams on GitHub:
+
+ - [@kubernetes/sig-docs-maintainers](https://github.com/orgs/kubernetes/teams/sig-docs-maintainers)
+ - [@kubernetes/sig-docs-pr-reviews](https://github.com/orgs/kubernetes/teams/sig-docs-pr-reviews)
+
+Each can be referenced with their `@name` in GitHub comments to communicate with
+everyone in that group.
+
+These teams overlap, but do not exactly match, the groups used by the automation
+tooling. For assignment of issues, pull requests, and to support PR approvals,
+the automation uses information from OWNERS files.
+
+### OWNERS files and front-matter
+
+The Kubernetes project uses an automation tool called prow for automation
+related to GitHub issues and pull requests. The
+[Kubernetes website repository](https://github.com/kubernetes/website) uses
+two [prow plugins](https://github.com/kubernetes/test-infra/tree/master/prow/plugins):
+
+- blunderbuss
+- approve
```

Comment:
`Website` lets us use the `/website` command.

```suggestion
- approve
- website
```

Reply:
Looking at the prow plugins, I think this entire section will need to be revised.

The actual `k/website` enabled ones are (https://github.com/kubernetes/test-infra/blob/master/config/prow/plugins.yaml#L725):
```
  kubernetes/website:
  - milestone
  - project
```

`k/*` enables:
```
  kubernetes:
  - approve
  - assign
  - blunderbuss
  - cat
  - cla
  - dog
  - golint
  - goose
  - heart
  - help
  - hold
  - invalidcommitmsg
  - label
  - lgtm
  - lifecycle
  - owners-label
  - pony
  - retitle
  - shrug
  - sigmention
  - size
  - skip
  - slackevents
  - trigger
  - verify-owners
  - welcome
  - wip
  - yuks
```

Maybe there's a different way to get the point across?

/honk
/woof
/pony

Reply:
Telling readers about at least the top 5 or 6 commands is going to be really useful.